samples/fabric/fabric-cli/src/data/event_sub_sample.json

```diff
@@ -27,16 +46,35 @@
             "transactionFunc": "create"
         },
         "event_publication_spec": {
-            "ccArgs": [],
+            "ccArgs": ["keyClass", ""],
```

Comment:
What does "keyClass" denote?

Reply:
It's just a name for the key imported from remote network.